examples/05-payments-history.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Example 05 - How to retrieve your payments history.
+ */
+
+const express = require('express');
+const mollie = require('@mollie/api-client');
+
+const app = express();
+const mollieClient = mollie({ apiKey: 'test_buC3bBQfSQhd4dDUeMctJjDCn3GhP4' });
+
+app.get('/', (req, res) => {
+  let paymentParameters = {
+    limit: 5,
+  };
+
+  if (req.query.from) {
+    paymentParameters = {
+      ...paymentParameters,
+      from: req.query.from,
+    };
+  }
+
+  mollieClient.payments
+    .all(paymentParameters)
+    .then(payments => {
+      res.send(`
+        <ul>
+          ${payments
+            .map(
+              payment => `<li>
+                  Payment ID: ${payment.id}<br />
+                  Method: ${payment.method}<br />
+                  Amount: ${payment.amount.value} ${payment.amount.currency}<br />
+                  Created at: ${new Date(payment.createdAt)}
+                </li>`,
+            )
+            .join('')}
+
+            <br /><br />
```

Comment:
It's not valid HTML to nest `<br>` and `<a>` tags under `<ul>`. Please close the list first.